src/Paket.LoadingScripts/LoadingScriptsGenerator.fs

```diff
@@ -225,11 +110,69 @@ module ScriptGeneratingModule =
 
       dependencies
       |> Map.map (fun groupName packages ->
-        let packagesOrGroupFolder =
+        let packagesOrGroupFolder, groupName =
           match groupName.GetCompareString () with
-          | "main"    -> packagesFolder 
-          | groupName -> Path.Combine(packagesFolder.FullName, groupName) |> DirectoryInfo
-        generateFSharpScripts dependenciesFile lockFile packagesOrGroupFolder packages
+          | "main"    -> packagesFolder, None
+          | groupName -> Path.Combine(packagesFolder.FullName, groupName) |> DirectoryInfo, Some groupName
+        generateScripts getScriptName scriptGenerator framework dependenciesFile packagesOrGroupFolder groupName packages
         )
       |> ignore
-      
\ No newline at end of file
+
+  let generateFSharpScript (input: ScriptGenInput) =
+    let packageName = input.PackageInstallModel.PackageName
+    let packageFolder =
+      Path.Combine (input.PackagesOrGroupFolder.FullName, packageName.GetCompareString())
+      |> DirectoryInfo
+
+    let depLines =
+      input.DependentScripts
+      |> Seq.map (fun s -> sprintf "#load \"../%s\"" (s.Replace("\\", "/")))
```

Comment:
I think we need a helper function given a (assembly: FileInfo) and a (packageFolder: DirectoryInfo) generate a system agnostic path.

those s.Replace here and there seems hackish.